app/api/server/v1/chat.js

```diff
@@ -360,6 +360,30 @@ API.v1.addRoute('chat.ignoreUser', { authRequired: true }, {
 	},
 });
 
+API.v1.addRoute('chat.getIgnoredUsers', { authRequired: true }, {
+	get() {
+		const { rid } = this.queryParams;
+		if (!rid || !rid.trim()) {
+			throw new Meteor.Error('error-room-id-param-not-provided', 'The required "rid" param is missing.', { method: 'getIgnoredUsers' });
+		}
+
+		const { userId } = this.userId;
+		if (!userId) {
+			throw new Meteor.Error('error-invalid-user', 'Invalid user', { method: 'getIgnoredUsers' });
+		}
+
+		const subscription = Subscriptions.findOneByRoomIdAndUserId(rid, userId);
+		if (!subscription) {
+			throw new Meteor.Error('error-invalid-subscription', 'Invalid subscription', { method: 'getIgnoredUsers' });
+		}
+
+		const result = subscription.ignored;
```

Comment:
This variable isn't necessary, you can pass the subscription.ignore directly above